.github/workflows/ex_doc_version_check.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: ex_doc_version_check
+on:
+  schedule:
+    - cron: '0 0 * * *'  # Run daily at midnight
+
+
+jobs:
+  test:
+    name: Check ex_doc version
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Get current version
+        run: |
+            CURR=`curl -s https://api.github.com/repos/elixir-lang/ex_doc/tags?per_page=100 | \
+                  grep "\"name\":" | \
+                  sort --reverse | \
+                  head -n 1 | \
+                  awk '{print substr($2, 2, length($2)-3)}'`
```

Comment:
```suggestion
                  jq -r ".[0].name"
```

Reply:
`jq` is already installed in the Ubuntu runners...

Reply:
As-was, though, this was returning `v0.9.0`, most probably due to the `sort`. Maybe since the list returns the latest tag as the first tag we should just get `per_page=1` (?)